Dockerfile

```diff
@@ -5,17 +5,24 @@ FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
 
 # >> START Install base software
 
+RUN apt-get update && apt-get install -y \
+    software-properties-common \
+    curl
+
+RUN add-apt-repository ppa:git-core/ppa && \
+    curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
```

Comment:
No way we do that

Reply:
@cfreemoser please use packages and avoid curl | bash

Reply:
Well isn't that the recommended way to install git-lfs according to the [docs](https://github.com/git-lfs/git-lfs/wiki/Installation)?

However, if you think checking it out from GitHub is not a use case for this dockerfile I would be fine to completely remove this.

Reply:
It's in the docs but `sudo curl | bash` is never a good idea

Reply:
https://packages.ubuntu.com/search?keywords=git-lfs

Reply:
Well, I guess without the script git-lfs is not working? But I totally get your point. Since its actually just necessary whenever you want to clone the DeepSpeech Repo instead of copy it locally I moved it down and uncomment it. So it's just a hint if someone needs it - as I did.